apps/fluent-tester/src/E2E/Menu/pages/MenuPageObject.ts

```diff
@@ -78,6 +74,10 @@ class MenuPageObject extends BasePage {
     return await this._primaryComponent;
```

Comment:
driveby comment -- if this function always returns `await this._primaryComponent`, can we simplify or correct its purpose/add a use case?